index.css

```diff
@@ -133,4 +135,36 @@
     align-items: center;
     justify-content: center;
     font-weight: 100;
-}
\ No newline at end of file
+}
+.sidebar {
+    width: 240px;
+    background-color:var(--darkwhite);
+    border-right: 1px solid var(--white);
+    height: 100vh; 
+    display: flex;
+    align-items: flex-start;
+    width: fit-content;
+  }
+  
+  
+  .sidebar__time-tracker {
+    display: flex;
+    align-items: center;
+    background-color:#e9ecef;
```

Comment:
use variables, apply throughout the file

Reply:
colors have been stored in variables